packages/node-cli/preinstall.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const readline = require('readline');
+const os = require('os');
+const RED_COLOR = '\x1b[31m';
+const COLOR_RESET = '\x1b[0m';
+const message = 'The installation will download the SuiteCloud SDK runtime dependency\n' +
+	'for this package. By downloading the SuiteCloud SDK dependency,\n' +
+	'you are accepting the Oracle Free Use Terms and Conditions license\n' +
+	'displayed above.';
+const REJECT_EXIT_CODE = 1;
+
+function showLicense() {
+
+	const rl = readline.createInterface({
+		input: process.stdin,
+		output: process.stdout,
+	});
+
+	rl.on('close', () => {
+		console.log(`${os.EOL}Installation aborted by user.`);
+		process.exit(REJECT_EXIT_CODE);
+	});
+
+	console.log(RED_COLOR);
```

Comment:
I suppose we cannot use chalk because in here. Could it be possible to put RED_COLOR, message and COLOR_RESET in the same console.log?

Not a big deal if it's not possible.